benchmarks/Cargo.toml

```diff
@@ -13,6 +13,10 @@ csv = "1.1.6"
 [target.'cfg(target_os = "linux")'.dependencies]
 jemallocator = "0.3.2"
 
+[target.'cfg(target_os = "macos")'.dependencies]
+mimalloc = { version = "0.1.29", default-features = false }
+
+
```

Comment:
```suggestion
```

Reply:
I am also happy with using mimalloc everywhere. I wanted to make a minimal change in the first place, which is why I kept jemalloc on Linux. Using jemalloc everywhere wouldn't be possible I think, since according to https://lib.rs/crates/jemallocator , some jemalloc tests don't pass on macOS.

Reply:
What I can do is:
1. Amend the PR to unconditionally use mimalloc as the global allocator everywhere
2. Run the benchmarks

Then:
1. If the benchmarks did not regress, merge the PR, then use mimalloc on the meilisearch repo as well
2. If the benchmarks regressed, only use mimalloc on macOS

Does that sound good?

Reply:
Yup, sound very good to me :)